vendor/bat-native-ledger/src/bat/ledger/internal/grants.h

```diff
@@ -44,6 +46,7 @@ class Grants {
       ledger::GetGrantCaptchaCallback callback);
 
   void GetGrantsCallback(
+      std::string safetynet_token,
```

Comment:
`const`

Reply:
I guess it is passed from Bind, so we cannot use reference as it crashes there.

Reply:
`GetGrantsCallback` is a callback of  `LoadURL`  from `Grants::FetchGrants`. At the moment when it's called, the  SafetyNet's `result_string` is gone. So we need it by value.